deps/src/serial_relay.h

```diff
@@ -0,0 +1,37 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+
+#include <libserialport.h>
+#include <zmq.h>
+
+#define DEBUG 1
```

Comment:
It'd be better to make this a CMake variable so the user can more easily control it from the build system.